### DISCUSSION TITLE
element 中的el-upload组件选择视频,Firefox 、Google Chrome浏览器,QQ浏览器本地预览没有显示

### DISCUSSION THREAD

**Discussion by xiaodi689** [element-plus/element-plus#18476] [Q&A]
element 中的el-upload组件选择视频,Firefox 、Google Chrome浏览器,QQ浏览器本地预览没有显示

**Comment by xiaodi689** [comment]
测试存在这个问题：https://element.eleme.cn/#/zh-CN/component/upload

**Reply by Yorlg** [reply]
你想要回显直接用slot不就好了，跟ui组件这个有啥关系
